### PR TITLE
[Enhancement] Not showing JDBC external table password

### LINKED ISSUE #28230: Not showing password on show resouces
### Steps to reproduce the behavior (Required)
show resources
### Expected behavior (Required)
password not mask
image

### Real behavior (Required)
 not show password
### StarRocks version (Required)
SR3.X

### PR DESCRIPTION
**Pull request by letian-jiang**
Fixes #28230

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5
  - [x] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 2 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/JDBCResource.java|0|2|00.00%|[136, 137]|

File: fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java [hunk 1]
```diff
@@ -133,6 +133,9 @@ public String getProperty(String propertyKey) {
     protected void getProcNodeData(BaseProcResult result) {
         String lowerCaseType = type.name().toLowerCase();
         for (Map.Entry<String, String> entry : configs.entrySet()) {
+            if (entry.getKey().equalsIgnoreCase(PASSWORD)) {
```
Comment (kevincai) [inline fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java:136]: do we skip the config or output something but masked information, e.g. replace the real password with `******`

Comment (letian-jiang) [reply fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java:136]: I think skipping is simpler and better.

**Review by kevincai — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/JDBCResource.java|2|2|100.00%|[]|

**Review by stephen-shelby — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.4

**Pull request merged**